src/ALZ/Private/Test-ALZGitRepository.ps1

```diff
@@ -13,9 +13,14 @@ function Test-ALZGitRepository {
         return $true
     }
     $gitInit = Read-Host "Initialize the directory $alzEnvironmentDestination as a git repository? (y/n)"
-    if ($gitInit -ieq "y"  -and $PSCmdlet.ShouldProcess("gitrepository", "initialize")) {
-        git init $alzEnvironmentDestination
-        return $true
+    if ($gitInit -ieq "y" -and $PSCmdlet.ShouldProcess("gitrepository", "initialize")) {
```

Comment:
As discussed. Consider adding a second prompt for branch name that defaults to `main`.